### ISSUE TITLE
download testdata broken

### ISSUE THREAD

**Issue by shusso** [gcarq/freqtrade#607]
branch: develop, ccxt
```
(.env) [freqtrade] > python3 freqtrade/tests/testdata/download_backtest_data.py
Traceback (most recent call last):
  File "freqtrade/tests/testdata/download_backtest_data.py", line 11, in <module>
    parser = misc.common_args_parser('download utility')
AttributeError: module 'freqtrade.misc' has no attribute 'common_args_parser'
```

#230

**Comment by enenn** [comment]
It should probably also be moved into the `scripts` directory

**Comment by xmatthias** [comment]
I don't think this issue should not have the ccxt label as the same bug happens on develop (from looking at the error-msg, I think it was introduced by the objectify work).

Agree with @enenn, this seems to be a good candidate for `scripts`

**Comment by gcarq** [comment]
fixed on `feat/objectify-ccxt`, but we should keep this issue open until it is merged into develop.

**Comment by gcarq** [comment]
Fixed on develop with 5b25ed9
